meilisearch/models/task.py

```diff
@@ -1,32 +1,28 @@
 from __future__ import annotations
 
-from typing import Any, Dict, Optional
+from typing import Any, Dict, Union
 
 from camel_converter.pydantic_base import CamelBase
 
 
 class Task(CamelBase):
     uid: str
-    index_uid: str
+    index_uid: Union[str, None]
     status: str
     type: str
-    details: Optional[Dict[str, Any]]
+    details: Dict[str, Any]
```

Comment:
```suggestion
    details: Dict[str, Any]
```

The error field is missing, but lets wait for the bump where you will change it anyway